iis3dwb_reg.c

```diff
@@ -50,9 +50,7 @@ int32_t __weak iis3dwb_read_reg(stmdev_ctx_t *ctx, uint8_t reg,
                                 uint8_t *data,
                                 uint16_t len)
 {
-  int32_t ret;
-
-  ret = ctx->read_reg(ctx->handle, reg, data, len);

```

Comment:
Maybe we can do:

```
return ctx->read_reg(ctx->handle, reg, data, len);
```

what do you think?

Reply:
I agree

Reply:
done